test/syscall/syscall.py

```diff
@@ -628,7 +628,12 @@ def run(self):
         errfile = open(self.errfilename, 'w')
         if self.args.verbose:
             print('RUNNING: ' + str(callargs))
-        subret = subprocess.call(callargs, stdout=outfile, stderr=errfile)
+        # We have to overload the WIREDTIGER_CONFIG to ensure we perform
+        # fdatasync operations on checkpoints
+        my_env = os.environ.copy()
+        my_env["WIREDTIGER_CONFIG"] = "checkpoint_sync=true"
```

Comment:
@daveh86, I think it would be better to set `wiredtiger_open` options in the program that is running, that is `test/syscall/wt2336_base/main.c`.  Then if we wanted to try other durability or checkpoint options, we could do that by making other programs (e.g. additional subdirectories under test/syscall) .  As much as possible, syscall.py should not know about WiredTiger, it should just be a pattern matching program.